tests/helpers/userAndOrg.ts

```diff
@@ -4,12 +4,12 @@ import { nanoid } from "nanoid";
 import type { Document } from "mongoose";
 
 export type TestOrganizationType =
-  | (InterfaceOrganization & Document<any, any, InterfaceOrganization>)
-  | null;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (InterfaceOrganization & Document<any, any, InterfaceOrganization>) | null;
 
 export type TestUserType =
-  | (InterfaceUser & Document<any, any, InterfaceUser>)
-  | null;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Remove. We need properly linted code.

Reply:
If I remove it from here then I have to edit all test files because each file have used 'any' type which would be out of scope of this pr shall I proceed to change all the 'any' type ??

Reply:
Or @palisadoes  sir I would solve this another pr ?

Reply:
You would only have to edit the test files submitted in this PR correct? That is in scope of this PR.
We prefer not to merge code without proper tests and linting that is why we have automation to verify compliance with our goals

Reply:
But sir I have to as in most of the test files there is use of type of any I have to edit all those files, if not then it shows linting error

Reply:
1. We have created a linter that only lints on committed files.
2. This was merged about a week ago
3. Please merge your code with the latest upstream to see whether it is in effect for you

Reply:
Yes sir as their are many files which uses testUser and testOrganization therefore if I change type any of testUser and testOrganization I have to fix type in every file which uses these two and that will be a lot of files shall I do that ?

Reply:
1. I see, it's hundreds of files. 
2. We'll need to fix this some other way